### PR TITLE
CF-975 updated Atom Content Element Example 

### PR DESCRIPTION
**Pull request by stanzikratel**
This addresses a Disqus comment about the example being incorrect

### CODE REVIEW

File: src/docbkx/samples/AtomContentElementExample.xml [hunk 1]
```diff
@@ -1,6 +1,28 @@
-<atom:content type="application/xml"
-              xmlns:atom="http://www.w3.org/2005/Atom">
-    <event xmlns="http://docs.rackspace.com/core/event" xmlns:sample="http://docs.rackspace.com/usage/widget/explicit" dataCenter="DFW1" endTime="2013-03-15T23:59:59Z" environment="PROD" id="e53d007a-fc23-1131-975c-cfa6b29bb814" region="DFW" resourceId="4a2b42f4-6c63-11e2-815b-7fcbcf67f549" startTime="2013-03-15T13:51:11Z" tenantId="1234" type="USAGE" version="1">
-        <sample:product dateTime="2013-09-26T15:32:00Z" disabled="false" enumList="BEST BEST" label="sampleString" mid="6e8bc430-9c3a-11d9-9669-0800200c9a66" num_checks="1" resourceType="WIDGET" serviceCode="Widget" stringEnum="3.0.1" time="15:32:00Z" version="1"/>
+<?xml version="1.0" encoding="UTF-8"?>
+<content>
```
Comment (shintasmith) [inline src/docbkx/samples/AtomContentElementExample.xml]: This probably needs to be:
```
<content xmlns="http://www.w3.org/2005/Atom">
```

File: src/docbkx/samples/AtomContentElementExample.xml [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+            <stringEnum>3.0.1</stringEnum>
+            <time>15:32:00Z</time>
+            <version>1</version>
+        </product>
+        <region>DFW</region>
+        <resourceId>4a2b42f4-6c63-11e2-815b-7fcbcf67f549</resourceId>
+        <startTime>2013-03-15T13:51:11Z</startTime>
+        <tenantId>1234</tenantId>
+        <type>USAGE</type>
+        <version>1</version>
```
Comment (usnavi) [inline src/docbkx/samples/AtomContentElementExample.xml]: I'm confused.  This doesn't look like a product schema node.  @stanzikratel swing by and lets discuss.

File: src/docbkx/samples/AtomContentElementExample.xml (hunk 1 above)
Comment (stanzikratel) [inline src/docbkx/samples/AtomContentElementExample.xml]: @shintasmith Thanks for the feedback. I made the update

File: src/docbkx/samples/AtomContentElementExample.xml (hunk 2 above)
Comment (stanzikratel) [inline src/docbkx/samples/AtomContentElementExample.xml]: @usnavi I went ahead and replaced this with one of the message samples for autoscale to ensure that we have a good product schema node example. I'll show you tomorrow.

**Comment by VinnyQ** [comment]
Constanze, you will also need to update the following file:
src/docbkx/feeds-getting-started.xml
Line #945 in this para block
```
<para>
    <example>
        <title>Atom content element - XML example</title>
        <programlisting language="xml"><xi:include href="./samples/AtomContentElementExample.json" parse="text"/></programlisting>
    </example>
</para>
```

And change "./samples/AtomContentElementExample.json" to "./samples/AtomContentElementExample.xml"

**Comment by VinnyQ** [comment]
FYI, this section 6.1.3 Atom content element section in the Getting Started guide seems to be the mirror of the 2.4.1.3 Atom content element section in the API Developer Guide here:  http://docs.rackspace.com/cloud-feeds/api/v1.0/feeds-devguide/content/Atom_Content-Element.html

Should we make them both the same examples after we determined to correct xml and json content this section should have?

**Comment by stanzikratel** [comment]
@VinnyQ thanks for the feedback. I will make sure that both guides point to the exact same samples.

**Comment by stanzikratel** [comment]
@shintasmith and @usnavi Updated both JSON and XML file with message example that Greg and I discussed

**Comment by shintasmith** [comment]
looks good to me, I will let @usnavi or @VinnyQ review and merge

File: src/docbkx/samples/AtomContentElementExample.xml [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+           version="1">
+        <cb-store:product
+           resourceType="AGENT"
+           serverID="9445"
+           serviceCode="CloudBackup"
+           storage="99992827882"
+           version="1" />
     </event>
-</atom:content>
+    </content>
```
Comment (usnavi) [inline src/docbkx/samples/AtomContentElementExample.xml]: ok, this is super picky, but might as well fix it here.  Can you remove the indenting?

@shintasmith  - I've worked on so much puppet i've become an ops guy!

Comment (shintasmith) [inline src/docbkx/samples/AtomContentElementExample.xml]: we are all, @usnavi, we are all.

**Comment by VinnyQ** [comment]
@stanzikratel samples look good.  You still need to fix the the file src/docbkx/feeds-getting-started.xml, line number 945 where it is listing the file for <programlisting language="xml">.  That needs to point to the .xml sample, but currently is pointing to the .json sample file (hence the reason why the user is seeing json instead of xml in their comment).

**Comment by stanzikratel** [comment]
@VinnyQ That was an easy fix :-) I went ahead and updated the GS guide.

**Comment by usnavi** [comment]
lgtm - @VinnyQ  can merge when he's happy with it & the build passes.

**Pull request merged**